code_samples/logging.js

```diff
@@ -19,59 +19,55 @@ var Config = require('hazelcast-client').Config;
 var HazelcastClient = require('hazelcast-client').Client;
 var LogLevel = require('hazelcast-client').LogLevel;
 
-if (process.argv.length != 3) {
-    console.log('Run as node logging.js [logger]');
-    console.log('[logger]: winston/default');
-} else {
-    var cfg = new Config.ClientConfig();
+var cfg = new Config.ClientConfig();
 
-    if (process.argv[2] === 'winston') {
-        var winstonAdapter = {
-            logger: new (winston.Logger)({
-                transports: [
-                    new (winston.transports.Console)()
-                ]
-            }),
+var winstonAdapter = {
+    logger: new (winston.Logger)({
+        transports: [
+            new (winston.transports.Console)()
+        ]
+    }),
 
-            levels: [
-                'error',
-                'warn',
-                'info',
-                'debug',
-                'silly'
-            ],
+    levels: [
+        'log',
+        'off',
+        'error',
```

Comment:
ordering of these should follow the order of the `LogLevel`
For example, `debug` must have the index of 3 (value of `LogLevel.DEBUG`)